releases.yml

```diff
@@ -46,6 +46,11 @@ releases:
             is:
               nvr: ose-tools-container-v4.12.0-202405080507.p0.gd691257.assembly.stream.el8
           why: Query from assembly basis event failed to replicate referenced nightly content exactly. Pinning to replicate.
+        - distgit_key: ironic-static-ip-manager
+          metadata:
+            is:
+              nvr: ironic-static-ip-manager-container-v4.12.0-202405091406.p0.ga8ade8f.assembly.4.12.57.el9
+          why: Query from assembly basis event failed to replicate referenced nightly content exactly. Pinning to replicate.
```

Comment:
Let's mention the real reason to do this .. i think we have a ticket related to signing that we opened? if not a one sentence summary of why rpm in this container image are unshippable due to brew issue